lsp-mode.el

```diff
@@ -5767,7 +5767,12 @@ REFERENCES? t when METHOD returns references."
   (let ((params (if args
                     (list :command command :arguments args)
                   (list :command command))))
-    (lsp-request "workspace/executeCommand" params)))
+    (condition-case-unless-debug err
+        (lsp-request "workspace/executeCommand" params)
+      (error
+       (lsp--error "Please open an issue in lsp-mode for implementing `%s'.\n\n%S"
```

Comment:
What if server return an error on `workspace/executeCommand`?

Reply:
We will throw other error message for that. But yes, it should be handle differently.

Reply:
On other thought, when server fail to handle `workspace/executeCommand`, we should still ask user open a bug for client handle that command.
Server may choose to only handle some command with `workspace/executeCommand`

Reply:
I see, thanks